tests/vppoutputencode.h

```diff
@@ -23,17 +23,48 @@
 #define vppoutputencode_h
 #include "VideoEncoderHost.h"
 #include "encodeinput.h"
+#include <string>
 #include <vector>
 
 #include "vppinputoutput.h"
+using std::string;
+//#include "yamitranscodehelp.h"
 
+class EncodeParams
+{
+public:
+    EncodeParams();
+
+    VideoRateControl rcMode;
+    int32_t initQp;
+    int32_t bitRate;
+    int32_t fps;
+    int32_t ipPeriod;
+    int32_t ipbMode;
+    int32_t kIPeriod;
+    uint32_t fourcc;
+};
+
+class CmdLineParams
```

Comment:
A little strange to use this name, since CmdLineParams is not only used to parse command parameters. Can we rename it to be TranscodeParams for example.